docs/tutorial-modifier-roles/add-role.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 3
+---
+
+# Add Role
+
+After confirming the transaction, you should return to the Zodiac App, where your configured Role Modifier will be displayed in the left side panel. Click anywhere on the modifier's card to access the Roles App.
+
+![Roles App - zero state](/img/tutorial/roles_03.png)
+
+## Create a Role
+
+![Roles App - zero state](/img/tutorial/roles_04.png)
+Click on the "Create a Role" button to open the Role view where you can add members and targets to the role. The Role name is created automatically.
+
+Once you've added the appropriate members and targets to this role (see below), you can click "Create Role". Only then will your changes to the role be saved. Note that you must connect your wallet (and be a key signer in connected safe) in order to create the role.
+![Create Role view](/img/tutorial/roles_05.png)
+
+## Members
+
+![Roles App - Add a Member modal](/img/tutorial/roles_06.png)
+
+Members are accounts that that the role is assigned to. Once an account is assigned the role, it can call the targets on behalf of the avatar by passing transactions to the avatar for execution (via the Roles Modifier).
+
+## Targets
+
+![Roles App - Add a Target modal](/img/tutorial/roles_07.png)
+
+Targets are the accounts that the members can interact with on behalf of the avatar.
```

Comment:
Not sure if `accounts` is the right term for `Targets`... could we say "Targets are the contracts...."? What do you think?

Reply:
I agree that it sounds wired. However, an externally owned account can also be a target, not just contract accounts. Therefore, technically speaking, I think the term is correct. But I agree that it can sound wired, so it might not be the most user-friendly.